README.md

```diff
@@ -18,25 +17,56 @@ var favoriteCharacter = "Jon Snow"
+
+The type of a variable is important, because it can only hold values of the same type, even if you change the value. `favoriteCharacter` must always be a string—it cannot change to an `Int` or a `Float`.
+
+Take a look at the [playground](Variables.playground) for this lesson. You can see how a variable in Swift is declared, and how you can change the value. Try it for yourself! Change `favoriteCharacter` to _your_ favorite character and see what happens.
+
+## Constants
+
+Swift offers a second way to associate a name with a value: a _constant_. Like a variable, a constant consists of a name and an associated value. Unlike a variable, however, the value of a constant cannot change over time; once you _initialize_ a constant (declare it and set it to a value), you cannot change it. Ever.
+
+A Swift constant is declared like a variable, except it uses the keyword `let` instead of `var`:
+
+```
+let ultimateFavoriteCharacter = "Arya Stark"
+```
+
+From here on out, `ultimateFavoriteCharacter` will always have the value "Arya Stark". If you try to change it:
+
+```
+ultimateFavoriteCharacter = "Daenerys Targaryen"
+```
+
+You will get an error from the Swift compiler. Check out the [playground](Variables.playground) to see this for yourself. You will see an error issued on line 10.
+
+Like variables, constants also have a type. Since you cannot change the value of a constant, you won't have to worry about changing the value to a different type. However, the type of constants still matters, since it dictates how you can interact with and use the constant, as you'll learn as you progress through your Swift studies.
+
+## Using Variables and Constants
+
+Why use variables and constants in your program? Both variables and constants are useful when you want to associate a name that never changes with a value that may change. You can then refer to that variable by a constant name. As shown in the playground associated with this lesson, once you associate the name `favoriteCharacter` with a value ("Jon Snow", for example), you can use `favoriteCharacter` to refer to that value for the rest of the program, such as when you pass it to functions like `print()`. (You will learn more about functions in a future lesson.)
 
+## Choosing Between Variables and Constants
 
+When should you use a variable, and when should you use a constant? A variable should be used when a value might change during the course of a single run of a program. For example, when you start running a program, your favorite may be Jon Snow, but perhaps by the end of the program's run, you may want to switch your allegiance to another character. A constant, on the other hand, is a way to bind a name to a value that won't change. This lets you refer to that value using a label without being concerned with the exact value that label takes.
```

Comment:
> ...you may want to switch your allegiance to another character.

After you mention that, maybe in parenthesis afterwards you could mention.. "Like we did above when we changed the value of our `favoriteCharacter` variable from "Jon Snow" to "Tyrion Lannister".

> This lets you refer to that value using a label without being concerned with the exact value that label takes.

I'm thinking about simplifying this sentence a little. Maybe just state something like this.. "This lets you refer to that value throughout your program where you are certain without a doubt that the value will *never* change."